### ISSUE TITLE
Database fields not written to _ide_helper_models.php

### ISSUE THREAD

**Issue by ionesculiviucristian** [barryvdh/laravel-ide-helper#1120]
### Versions:
- ide-helper Version: 2.8.2
- Laravel Version: 8.18.1
- PHP Version: 8.0.0
- MySQL Version: 8+

(using Docker and Laravel Passport, if it matters)
### Description:

When running ```php artisan ide-helper:models --nowrite``` the model ```User.php``` located in ```app/Models``` doesn't write the database fields, only the default properties/methods. 
In ```Barryvdh\LaravelIdeHelper\Console\ModelsCommand.php``` the lines:

```php
        if (strpos($table, '.')) {
            [$database, $table] = explode('.', $table);
        }
```
expect the ```$table``` to have the database name in but for me ```$table``` is ```'users'``` and as such returns here

```php
        $columns = $schema->listTableColumns($table, $database);

        if (!$columns) {
            return;
        }
```

**Comment by mfn** [comment]
@ionesculiviucristian is this a recent regression for your? Or did it newer work / new project?

If a regression, can you check out older ide-helper / Laravel versions?

thank you!

**Comment by ionesculiviucristian** [comment]
Tried with 2.8.1 (lowest with Laravel 8 support) and 2.8.2. Same result.

**Comment by mfn** [comment]
Your config looks exactly like mine, I'm on PHP 7.4.13 with Laravel 8.17.2 and ide-helper 2.8.1/2.8.2 but work with and without `--nowrite`.

**Comment by mfn** [comment]
Do you have doctrine/dbal installed?

**Comment by ionesculiviucristian** [comment]
@mfn No. i don't but for whatever reason, today, after doing a composer update, it just worked. I guess the issue can be closed.

I see this package requires ```"doctrine/dbal": "~2.3"```. Is that not for getting the database fields?

**Comment by mfn** [comment]
It is, basically doctrine/dbal is required for the feature you expected to work :}

Do you still feel we can close this? I'm not sure I entirely follow how it was suddenly fixed for your and we may still have something to improve?

**Comment by ossicoinc** [comment]
I can confirm this works - it doesn't support tables that are in a different schema than dbo

In model: 
`$table = 'sc.MyTable';`

Thanks for the workaround!

**Comment by SimplyCorey** [comment]
https://github.com/barryvdh/laravel-ide-helper/pull/1349 should fix the schema issue.

**Comment by developh** [comment]
I have tried to add schema to table name in model `__construct`
`
```
class Example extends Model
{
    public function __construct(array $attributes = [])
    {
        parent::__construct($attributes);

        $this->table = Schema::$bigWin . '.table_name';
    }
}
```
and it works nicely.
